### DISCUSSION TITLE
What is machine-learning algorithms?

### DISCUSSION THREAD

**Discussion by PeterZhang17** [PeterZhang17/HelloWorld#25] [Q&A] [answered]
What is machine-learning algorithms?

**Comment by neilwang0913** [comment]
clustering

**Comment by fatanehfarahani** [comment, marked as answer]
Linear Regression
SVM (Support Vector Machine)
KNN (K-Nearest Neighbors)
Logistic Regression
Decision Tree
K-Means
Random Forest
Naive Bayes
Dimensional Reduction Algorithm
Gradient Boosting Algorithms

**Comment by neilwang0913** [comment]
How about LLM now ❤️
